test/task/MonitorTask.test.ts

```diff
@@ -190,6 +192,9 @@ describe('MonitorTask', () => {
       new ConnectionHealthPolicyConfiguration(),
       new ConnectionHealthData()
     );
+    context.meetingSessionConfiguration = new MeetingSessionConfiguration();
```

Comment:
The default value of `attendeePresenceTimeoutMs` must be `0` (See the comment in `MeetingSessionConfiguration`), so this code won't work.

You can add a new group of tests using `describe` and `before` inside like the following:

```
describe('run attendeePresenceTimeoutMs > 0 ', () => {
  beforeEach(() => {
     context.meetingSessionConfiguration = new MeetingSessionConfiguration();
     context.meetingSessionConfiguration.attendeePresenceTimeoutMs = 1000;
     ...
  });

   it('your new tests here', () => {});
})
```

Reply:
Added the as separate group of tests for better readability.